plopfile.mjs

```diff
@@ -0,0 +1,162 @@
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
+
+const vectaraWebsiteQuestions = require("./config/website-vectara/queries.json");
+const vectaraDocsQuestions = require("./config/vectara-docs/queries.json");
+const askFeynmanQuestions = require("./config/ask-feynman/queries.json");
+
+const DEFAULT_CONFIGS = {
+  "vectara-docs": {
+    customerId: "1366999410",
+    corpusId: "1",
+    apiKey: "zqt_UXrBcnI2UXINZkrv4g1tQPhzj02vfdtqYJIDiA",
+    appTitle: "Vectara Docs Search",
+    questions: JSON.stringify(vectaraDocsQuestions.questions),
+  },
+
+  "vectara-website": {
+    customerId: "1366999410",
+    corpusId: "2",
+    apiKey: "zqt_UXrBcnnt4156FZqMtzK8OEoZqcR0OrecS5Bb6Q",
+    appTitle: "Vectara Website Search",
+    questions: JSON.stringify(vectaraWebsiteQuestions.questions),
+  },
+
+  "ask-feynman": {
+    customerId: "1366999410",
+    corpusId: "3",
+    apiKey: "zqt_UXrBclYURJiAW9MiKT1L60EJC6iaIoWYj_bSJg",
+    appTitle: "Ask Feynman",
+    questions: JSON.stringify(askFeynmanQuestions.questions),
+  },
+};
+
+let presetAppName;
+
+const checkNeedsConfiguration = (context) => {
+  return !context.presetAppName;
+};
+
+export default function (plop) {
+  plop.setGenerator("env", {
+    description: "Application Configuration Variables",
+    prompts: async function (inquirer) {
+      const acknowledgeAns = await inquirer.prompt({
+        type: "input",
+        name: "acknowledgePrimer",
+        message:
+          "Welcome to vectara-answer!\nLet's configure your application.\nYou can choose from a pre-configured one, or configure your own.\nNOTE: THIS WILL OVERWRITE EXISTING CONFIGS.\n\nPress Enter to continue.",
```

Comment:
When I read this, I wondered "What does 'overwrite existing configs' mean?" Can we update this prompt to be specific about overwriting the `.env` file?